QA/PoroDevTest/src/main/java/common/api_setup/api_common/UserDetailsGenerator.java

```diff
@@ -22,20 +24,125 @@ public static User generateRandomUserDetails() {
                 UserDetailsList.passSymbols.get(rn.nextInt(UserDetailsList.passSymbols.size()))+
                 rn.nextInt(10000) + 1);
         user.setDepartment(0);                                // will have to be added later
-        user.setRole(rn.nextInt(1));         // as it has to be 0 or 1
+        user.setRole(rn.nextInt(1));
         user.setPosition(UserDetailsList.userPosition.get(rn.nextInt(UserDetailsList.userPosition.size())));
         user.setAvatarUrl("string");   // will have to be added later
 
 
 
         return user;
     }
-/*
-Method for transforming object to JSON
- */
+
+    // Strings as a valid parts of jwt token
+
+    public static String validJwtHeader = "eyJhbGciOiJodHRwOi8vd3d3LnczLm9y" +
```

Comment:
All those variables should be final and their names should be will upper letters, for example VALID_JWT_HEADER

Reply:
This data is sensitive, so not sure it should be present in solution

Reply:
All data like this moved to final (be_registration.properties, be_runtime_data.properties)